doc/installation_instructions/site-install.rst

```diff
@@ -0,0 +1,17 @@
+.. _site install:
+
+Site Installation
+=================
+
+Certain HPE customer machines have a 'site installation' of SmartSim. This means
+that users can bypass the ``smart build`` step that builds the ML backends and
+the Redis binaries. Users on these platforms can install SmartSim from PyPi or
```

Comment:
Super pedantic suggestion: preferred abbreviation of "Python Package Index" is "PyPI" (capital I). Noticed this applies to a couple of files.

Reply:
Went through and changed the ones I could find.